tide_core.drush.inc

```diff
@@ -0,0 +1,188 @@
+<?php
+
+/**
+ * @file
+ * Drush file to define custom drush command redirect-import.
+ */
+
+use Drupal\redirect\Entity\Redirect;
+
+/**
+ * Implements hook_drush_command().
+ */
+function tide_core_drush_command() {
+  $items = [];
+
+  $items['redirect-import'] = [
+    'description' => "Imports a file and adds redirects to the redirect table",
+    'arguments' => [
+      'filepath' => 'Path to CSV (comma-separated), JSON or PHP file containing old and new URLs for redirects',
```

Comment:
Can this be a URI?

Reply:
Supports both, full URL is used for redirecting from or to external sites but if either is internal the entry would just specify the internal URI link.